python/paddle/v2/dataset/tests/common_test.py

```diff
@@ -32,6 +32,27 @@ def test_download(self):
             paddle.v2.dataset.common.download(
                 yi_avatar, 'test', 'f75287202d6622414c706c36c16f8e0d'))
 
+    def test_split(self):
+        def test_reader():
+            def reader():
+                for x in xrange(10):
+                    yield x
+            return reader
+        _, temp_path = tempfile.mkstemp()
+        paddle.v2.dataset.common.split(test_reader(), 4,
+            suffix=temp_path + '/test-%05d.pickle')
+        files = glob.glob(temp_path + '/test-%05d.pickle')
+        self.assertEqual(len(files), 3)
+
+    def test_cluster_file_reader(self):
+        _, temp_path = tempfile.mkstemp()
+        for x in xrange(5):
+            with open(temp_path + '/%05d.test' % x) as f:
+                f.write('%d\n' % y)
+        reader = paddle.v2.dataset.common.cluster_files_reader(
+            temp_path + '/%05d.test', 5, 0)
+        for idx, e in enumerate(reader()):
+            self.assertEqual(e, str("0"))
 
```

Comment:
It will spend so much time that downloads all the dataset, especially in China, so maybe we can just test the function simply.